assets/app/components/app.js

```diff
@@ -0,0 +1,53 @@
+
+import React from 'react';
+
+import { routeTypes } from '../constants';
+import store from '../store';
+
+import AddSite from './addSite';
+import SiteList from './siteList';
+import SiteContainer from './siteContainer';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    let state = this.props.state;
+    let navigation = this.props.state.navigation;
+    let content;
+
+    switch (navigation.name) {
```

Comment:
Wondering if you have used react-router before, and if so, what your thoughts are on it? I find its a really nice way to package up the routing abstractions and provides a clear view of the structure of your application.